plugins/vpp/l3plugin/vppcalls/vpp1904/dhcpproxy_dump.go

```diff
@@ -0,0 +1,46 @@
+package vpp1904
+
+import (
+	vpp_l3 "github.com/ligato/vpp-agent/api/models/vpp/l3"
+	"github.com/ligato/vpp-agent/plugins/vpp/binapi/vpp1904/dhcp"
+	"github.com/ligato/vpp-agent/plugins/vpp/l3plugin/vppcalls"
+	"net"
+)
+
+func (h *DHCPProxyHandler) DumpDHCPProxy() (*vppcalls.DHCPProxyDetails, error) {
+	var entry *vppcalls.DHCPProxyDetails
+
+	reqCtx := h.callsChannel.SendMultiRequest(&dhcp.DHCPProxyDump{})
```

Comment:
There is flag for IP protocol in `dhcp.DHCPProxyDump` request and you should actually run this request two times, for both IPv4 and IPv6.

https://github.com/ligato/vpp-agent/blob/400e6c3ae7ef39f78a15f8f1830f017fecce765b/plugins/vpp/binapi/vpp1904/dhcp/dhcp.ba.go#L447-L449